trailblazer/server/api.py

```diff
@@ -54,18 +55,17 @@ def analyses():
     """Display analyses."""
     per_page = int(request.args.get("per_page", 100))
     page = int(request.args.get("page", 1))
-    query = store.analyses(
-        status=request.args.get("status"),
-        query=request.args.get("query"),
-        is_visible=request.args.get("is_visible") == "true" or None,
+    analyses: Query = store.get_analyses_query_by_search_term_and_is_visible(
+        search_term=request.args.get("query"),
+        is_visible=bool(request.args.get("is_visible")),
```

Comment:
I'd place the extraction of the parameters above or in a separate function, matter of taste though. Feel free to disregard. Decreases the cognitive load a little to not have to understand the call and what the args are at once